src/config/locales.json

```diff
@@ -53,6 +54,12 @@
 				"github": "tesla1856"
 			}
 		],
+		"zh-cn":[
```

Comment:
Missing space after the `:`. Please be so kind and also fix the missing space on the `zh-tw` entry.

Reply:
Fixed.